BUILD.bazel

```diff
@@ -14,38 +14,7 @@ cc_library(
 
 go_library(
     name = "go_default_library",
-    srcs = [
-        "config.go",
-        "doc.go",
-        "engine.go",
-        "error.go",
-        "exporttype.go",
-        "extern.go",
-        "externtype.go",
-        "ffi.go",
-        "func.go",
-        "functype.go",
-        "global.go",
-        "globaltype.go",
-        "importtype.go",
-        "instance.go",
-        "linker.go",
-        "maybe_gc_no.go",
-        "memory.go",
-        "memorytype.go",
-        "module.go",
-        "shims.c",
-        "shims.h",
-        "slab.go",
-        "store.go",
-        "table.go",
-        "tabletype.go",
-        "trap.go",
-        "val.go",
-        "valtype.go",
-        "wasi.go",
-        "wat2wasm.go",
-    ],
+    srcs = glob(["**/*_test.go"]),
```

Comment:
This change belongs to the other set of srcs, not here.

Reply:
still not work QQ

Reply:
You are still missing some changes. May I suggest you remove the last commit and re-apply the content of the gist using `patch -p1 </path/to/gist` instead?

Reply:
Oh I didn't know that I can use patch. I kinda failed on apply it directly, but any I have successfully add it. Thank you. And I have added you as ao-author too

Reply:
That's great news. Thank you!